dask_expr/_rolling.py

```diff
@@ -221,10 +221,20 @@ class Rolling:
     """Aggregate using one or more operations
 
     The purpose of this class is to expose an API similar
-    to Pandas' `Rollingr` for dask-expr
+    to Pandas' `Rolling` for dask-expr
     """
 
-    def __init__(self, obj, window, groupby_kwargs=None, groupby_slice=None, **kwargs):
+    def __init__(
+        self,
+        obj,
+        window,
+        groupby_kwargs=None,
+        groupby_slice=None,
+        min_periods=None,
+        center=False,
+        win_type=None,
+        axis=0,
```

Comment:
let's remove this keyword completely

Reply:
Done and done https://github.com/dask-contrib/dask-expr/pull/497/commits/cbc057b0c961738e75f3f2d789f84bcab5268751